amundsen_application/static/js/components/common/Table/index.tsx

```diff
@@ -55,19 +73,26 @@ const ALIGNEMENT_TO_CLASS_MAP = {
   center: 'is-center-aligned',
 };
 
-type RowStyles = {
-  height: string;
-};
-
-type EmptyRowProps = {
-  colspan: number;
-  rowStyles: RowStyles;
-  emptyMessage?: string;
-};
-
 const getCellAlignmentClass = (alignment: TextAlignmentValues) =>
   ALIGNEMENT_TO_CLASS_MAP[alignment];
 
+const fieldIsDefined = (field, row) => row[field] !== undefined;
+
+const checkIfValidData = (
+  data: unknown[],
+  fields: string[]
+): data is ValidData[] => {
+  let isValid = true;
+
+  fields.forEach((field) => {
+    if (data.filter(fieldIsDefined.bind(null, field)).length === 0) {
+      isValid = false;
+    }
+  });
```

Comment:
just a nit for speed's sake. 

You might want to use a normal loop (I don't like them but in this case might be useful) to break when the false happens instead of keep iterating.  meaning, you want to break if anything turns false. 

for that. maybe 2 options. the aforementioned for loop so you can use break;

or in a functional way, you would have to flip the logic and use some instead of forEach.